ProcessLib/LIE/SmallDeformation/SmallDeformationProcess.cpp

```diff
@@ -129,14 +126,15 @@ SmallDeformationProcess<DisplacementDim>::SmallDeformationProcess(
     }
     for (std::size_t i = 0; i < vec_junction_nodeID_matIDs.size(); i++)
     {
-        std::vector<int> fracIDs;
-        for (auto matid : vec_junction_nodeID_matIDs[i].second)
-            fracIDs.push_back(
-                _process_data._map_materialID_to_fractureID[matid]);
-        auto* junction = createJunctionProperty(
-            i, *mesh.getNode(vec_junction_nodeID_matIDs[i].first), fracIDs);
-
-        _process_data._vec_junction_property.emplace_back(junction);
+        auto const& material_ids = vec_junction_nodeID_matIDs[i].second;
+        assert(material_ids.size() == 2);
+        std::array<int, 2> fracture_ids{
```

Comment:
warning:  suggest braces around initialization of subobject.
The use of double braces should fix the warning.

Reply:
Thanks, added in master.